projects/schematics/src/shared/utils/config-utils.ts

```diff
@@ -296,3 +300,60 @@ function convert(newValues: string | string[]): string {
   }
   return configValue;
 }
+
+export function getConfigs(sourceFile: SourceFile): Node[] {
+  const configs: Node[] = [];
+  const module = getModule(sourceFile);
+  if (!module) {
+    return [];
+  }
+  const literal = module.getFirstDescendantByKind(
+    tsMorph.SyntaxKind.ObjectLiteralExpression
+  );
+  if (literal) {
+    const properties = literal.getChildrenOfKind(
```

Comment:
Let's try to minimize the amount of indentations :)
We could do `const properties = literal.getChildrenOfKind() ?? []` and avoid the `if (properties) {`.